spectrocloud/resource_cluster_libvirt.go

```diff
@@ -509,31 +509,33 @@ func resourceClusterVirtUpdate(ctx context.Context, d *schema.ResourceData, m in
 
 		for _, mp := range ns {
 			machinePoolResource := mp.(map[string]interface{})
-			name := machinePoolResource["name"].(string)
-			if name == "" {
-				continue
-			}
-			hash := resourceMachinePoolLibvirtHash(machinePoolResource)
+			if machinePoolResource["name"].(string) != "" {
```

Comment:
```suggestion
                        // since known issue in TF SDK: https://github.com/hashicorp/terraform-plugin-sdk/issues/588
			if machinePoolResource["name"].(string) != "" {
```